README.md

```diff
@@ -1,8 +1,13 @@
 # falcon-integration-gateway [![Python Lint](https://github.com/CrowdStrike/falcon-integration-gateway/actions/workflows/linting.yml/badge.svg)](https://github.com/CrowdStrike/falcon-integration-gateway/actions/workflows/linting.yml) [![Container Build on Quay](https://quay.io/repository/crowdstrike/falcon-integration-gateway/status "Docker Repository on Quay")](https://quay.io/repository/crowdstrike/falcon-integration-gateway)
 
-Falcon Integration Gateway forwards findings from CrowdStrike Falcon platform to various [backends](fig/backends).
+Falcon Integration Gateway (FIG) forwards threat detection findings from CrowdStrike Falcon platform to the [backend](fig/backends) of your choice.
 
-Status: This is pre-release version (WIP).
+Learn more about CrowdStrike Falcon Threat Graph: [Product Page](https://www.crowdstrike.com/products/falcon-platform/threat-graph/), [Data Sheet](https://www.crowdstrike.com/resources/data-sheets/threat-graph/).
+
+Currently available backends are
+ * [AWS backend](fig/backends/aws) - pushes real-time threat detections to AWS Security Hub
+ * [Azure backend](fig/backends/azure) - pushes real-time threat detections to Azure Log Analytics
+ * [GCP Security Command Center](fig/backends/gcp) - pushes real-time threat detections to GCP Security Command Center
```

Comment:
```suggestion
 * [AWS Security Hub](fig/backends/aws) - pushes real-time threat detections to AWS Security Hub
 * [Azure Log Analytics](fig/backends/azure) - pushes real-time threat detections to Azure Log Analytics
 * [GCP Security Command Center](fig/backends/gcp) - pushes real-time threat detections to GCP Security Command Center
```

Reply:
Since the backends are named products, should they be called out by name? This was already done for the GCP backend.

Reply:
It was typo on my and to mention GCP Security Command Center.

I would like to have *backend* keyword on each line to help people understand the concept of backends.